### ISSUE TITLE
Website offline forever? ATL Server List claims 0% Uptime...

### ISSUE THREAD

**Issue by Zilacon** [ATLauncher/Website-Issues#39]
So my server has been online all day every day forever. I have 24 players online right now and your website is saying it has not been online at all for the last 4 days...

Can this be fixed please?
http://servers.atlauncher.com/server/ApocalypticGamingYogscastComplete/

Also the mod realms network seems to be affected by this same issue.

**Comment by RyanTheAllmighty** [comment]
Yup we're looking into it. Seems to be an issue with 1.6 servers only.

**Comment by Zilacon** [comment]
Alright thank you, could you reset everyone's uptime when this is fixed? Looks pretty bad for us.

**Comment by RyanTheAllmighty** [comment]
Yup as explained in the email sent out a couple weeks back, as were tinkering with the new system and fixing issues, we will be resetting the uptime constantly.

In the end the uptime will most likely just be removed from the site if we cannot fix the issue.

**Comment by Zilacon** [comment]
Alright, thank you for the quick response :)

**Comment by RyanTheAllmighty** [comment]
No problems. Sorry for the bad uptime, this was supposed to fix uptime issues yet made them worse which is always a good sign :\

**Comment by Zilacon** [comment]
Still an issue 11 days later...
My avg player base went from 25-30 players peak to 2-5 players peak.

No ones going to join a server listed as offline for 14 days and 0% uptime...

**Comment by Zilacon** [comment]
All other servers show as online but ours...
Our server has never been offline, ever. Why is it that we are the only ones listed for YCC thats offline?

**Comment by RyanTheAllmighty** [comment]
I believe it's because it's 1.6. We're most likely just going to remove uptime from the site completely until it's fixed

**Comment by Zilacon** [comment]
SeriousServers and ModRealms are 1.6... And they are online...

**Comment by Zilacon** [comment]
I have tried to change everything from every config i could, hell i even tried to change my port to another server thats online as well.

Not one of my servers will "connect" to ATL. I have disabled ALL firewalls on my entire network and my own dedicated servers, turned off dos protection, ping limits, etc. Nothing works.

Whenever i try to hit "Edit Server" it says "Cannot connect to this server" and on my console it says:
````
04-23 07:10:03 [INFO] [Minecraft] /46.101.51.137:33536 lost connection
04-23 07:10:29 [INFO] [Minecraft] /46.101.51.137:33937 lost connection
04-23 07:10:47 [INFO] [Minecraft] /46.101.51.137:33947 lost connection
````

A solution to this would be nice... Because it IS NOT on my end unless using MCPC is not allowed by ATL anymore.
